vars/createOpenshiftDockerSecret.groovy

```diff
@@ -0,0 +1,19 @@
+Closure call(String username, String password, String email) {
+    openshift.withCluster() {
+
+        if (openshift.selector('secrets').names().contains('secret/dockerhub')) {
+            openshift.selector('secret/dockerhub').delete()
+        }
+
+        openshift.secrets(
+            "new-dockercfg",
+            "dockerhub",
+            "--docker-server=docker.io/fhwendy",
```

Comment:
I think this should be a configurable since this is a shared library, also possibly the names "dockerhub" since it doesn't seem to be anything docker hub specific.